packages/core/useKeyModifierState/index.ts

```diff
@@ -1,13 +1,25 @@
 import { ref } from 'vue-demi'
-import { useEventListener } from '../useEventListener'
+import { useEventListener, WindowEventName } from '../useEventListener'
 
 export type Modifier = 'Alt' | 'AltGraph' | 'CapsLock' | 'Control' | 'Fn' | 'FnLock' | 'Meta' | 'NumLock' | 'ScrollLock' | 'Shift' | 'Symbol' | 'SymbolLock'
 
-const listenerEvents = ['keydown', 'keyup']
+const defaultEvents: WindowEventName[] = ['mousedown', 'mouseup', 'keydown', 'keyup']
 
-export function useKeyModifierState(modifier: Modifier) {
+export interface ModifierOptions {
+  /**
+   * Event names that will prompt update to modifier states
+   *
+   * @default ['mousedown', 'mouseup', 'keydown', 'keyup']
+   */
+  events?: WindowEventName[]
+}
+
+export function useKeyModifierState(modifier: Modifier, options: ModifierOptions = {}) {
+  const {
+    events = defaultEvents,
+  } = options
   const state = ref<null | boolean>(null)
-  listenerEvents.forEach((listenerEvent) => {
+  events.forEach((listenerEvent) => {
     useEventListener(document, listenerEvent, (evt: KeyboardEvent) => {
```

Comment:
We'd better make `document` configurable as well. See https://vueuse.org/guidelines.html#configurable-globals